configure

```diff
@@ -343,17 +343,30 @@ def b(value):
 
 
 def pkg_config(pkg):
-  cmd = os.popen('pkg-config --libs %s' % pkg, 'r')
-  libs = cmd.readline().strip()
-  ret = cmd.close()
-  if (ret): return None
-
-  cmd = os.popen('pkg-config --cflags %s' % pkg, 'r')
-  cflags = cmd.readline().strip()
-  ret = cmd.close()
-  if (ret): return None
-
-  return (libs, cflags)
+  pkg_config = 'pkg-config'
+  pkg_config = os.environ.get('PKG_CONFIG', pkg_config)
+  pkg_config = os.environ.get('PKG_CONFIG_PATH', pkg_config)
```

Comment:
I'll re-read the docs. A python library also added it to path which is why I considered it in the first place.